### PR TITLE
Add Engine - Flexi

### PR DESCRIPTION
**Pull request by wcfairytale**
Flexi Engine is a game engine developed by [**Snail Games**](https://store.steampowered.com/publisher/SnailGamesUSA/) with strong MMO ability. It was used in [Wushu Chronicles](https://store.steampowered.com/app/921390/Wushu_Chronicles/) and [Wushu Chronicles 2](https://store.steampowered.com/app/1465890/2_Wushu_Chronicles_2/).

### CODE REVIEW

**Comment by larsiusprime** [comment]
Hmmm, I think it's interesting to have matches for Chinese game engines especially and would like to know about more of them, but currently this particularly one only matches two games on all of SteamDB, and I'm not sure if that's enough, @xPaw ?

**Comment by xPaw** [comment]
The FX prefix is kinda generic, so I'm not sure. Are there more games outside of steam using this, will more come?

**Comment by wcfairytale** [comment]
for "The FX prefix is kinda generic"
- If fx_core.dll alone is too generic, will multi rule like "fx_core.dll and fx_gui.dll and fx_render.dll" work?
- In the README.md, I learned by []  we can have "or" rules，but I have not found out how how to make "and" rules yet.

for "Are there more games outside of steam using this"
- Yes, "Age of Wushu" for example, although it's been some years..

for "will more come?"
- Definitely, yes ;)

**Comment by xPaw** [comment]
I meant it's generic as it usually stands for "effects", I'll check if there are any other games we could possibly false positive with.

If not, I'll merge.

File: descriptions/Engine.Flexi.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Flexi Engine is a game engine developed by [**Snail Games**](https://store.steampowered.com/publisher/SnailGamesUSA/) with strong MMO ability. It was used in [Wushu Chronicles](https://store.steampowered.com/app/921390/Wushu_Chronicles/) and [Wushu Chronicles 2](https://store.steampowered.com/app/1465890/2_Wushu_Chronicles_2/).
```
[the remark this answers is not in the document]
Comment (wcfairytale) [reply descriptions/Engine.Flexi.md]: Done

**Pull request merged**